lib/engine/step/g_18_tn/buy_company.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require_relative '../buy_company'
+
+module Engine
+  module Step
+    module G18TN
+      class BuyCompany < BuyCompany
+        def can_buy_company?(entity)
+          companies = @game.purchasable_companies
+
+          entity == current_entity &&
+            (@game.turn == 1 || (@game.phase. available?('3') && @game.phase.buy_companies)) &&
```

Comment:
so you can buy companies if turn == 1 or available 3 and buy companies? i'm not exactly sure what's going on here

Reply:
By using event status, and introducing an event status for OR 1 buy company, I think the code became clearer.